project.godot

```diff
@@ -31,6 +31,12 @@ snap=false
 
 common/drop_mouse_on_gui_input_disabled=true
 
+[importer_defaults]
+
+texture={
+"flags/filter": false
+}
+
```

Comment:
This avoids us to change the import of every image to be scaled with nearest neighbour